src/Process/CpuCoreCounter.php

```diff
@@ -28,6 +29,17 @@ public function getNumberOfCpuCores(): int
 			return $this->count = 1;
 		}
 
+		// Support for Kubernetes pods with limited resources
+		// See: https://github.com/phpstan/phpstan/issues/7479
+		if (@is_file('/sys/fs/cgroup/cpu/cpu.cfs_quota_us') && @is_file('/sys/fs/cgroup/cpu/cpu.cfs_period_us')) {
```

Comment:
Since you do a file_get_contents() with `@` operator, you could remove the 2 is_file() calls beforehand and simplify the logic?

Reply:
It's consistent with the rest, please don't change that.

Reply:
Yeah, I did this based on the other conditions 🙂